packages/block-editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -9,7 +9,9 @@ import { orderBy } from 'lodash';
  */
 
 import { __ } from '@wordpress/i18n';
-import { Toolbar, Slot, DropdownMenu } from '@wordpress/components';
+import { Toolbar, Slot, DropdownMenu, createSlotFill } from '@wordpress/components';
+
+const { Slot: ToolbarControlsSlot } = createSlotFill( 'RichText.ToolbarControls' );
```

Comment:
This change was only to have a reference to the slot so we can pass it to SlotFillProvider right?

Reply:
> This change was only to have a reference to the slot so we can pass it to SlotFillProvider right?

Yes, rather than reference these by their string names (which so far we've considered an internal implementation detail), it seems we may instead want to reference the components themselves.